### PR TITLE
Add CV support to wand recharge pedestal

### PR DESCRIPTION
**Pull request by repo-alt**

### CODE REVIEW

File: src/main/java/com/mitchej123/hodgepodge/mixins/wandPedestalCV/MixinTileWandPedestal.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    @Shadow int counter = 0;
+
+    @Redirect(method ="Lthaumcraft/common/tiles/TileWandPedestal;updateEntity()V",
+        at=@At(value = "INVOKE",target="Lthaumcraft/common/items/wands/ItemWandCasting;getAspectsWithRoom(Lnet/minecraft/item/ItemStack;)Lthaumcraft/api/aspects/AspectList;"))
+    AspectList getAspectsWithRoomReplacement(ItemWandCasting wand, ItemStack wandstack)
+    {
+        AspectList as = wand.getAspectsWithRoom(wandstack);
+        if (as != null && as.size() > 0 && this.counter % 20 == 0) {
+            for(Aspect aspect : as.getAspects()) {
+                int drained = VisNetHandler.drainVis(worldObj, xCoord, yCoord, zCoord, aspect, 1);
```
Comment (Glease) [inline src/main/java/com/mitchej123/hodgepodge/mixins/wandPedestalCV/MixinTileWandPedestal.java:36]: Conceptually, 100CV (centi-vis) is 1Vis.

Comment (repo-alt) [reply src/main/java/com/mitchej123/hodgepodge/mixins/wandPedestalCV/MixinTileWandPedestal.java:36]: Huh, indeed. Ok, I'll have to accumulate it somwhere then

**Comment by Glease** [comment]
Wait a sec, it appears `ItemWandCasting.addRealVis()` is really `ItemWandCasting.addCentiVis()`, and that parameter actually means cvAmount not visAmount.

**Comment by repo-alt** [comment]
Ok, it turns out the wands store CV inside.

**Review by Glease — APPROVED** [review]
Looks fine

**Pull request merged**